src/Components/DashboardComponents/Profile.js

```diff
@@ -1,309 +1,345 @@
-import React, { useState, useEffect } from 'react'
-import styled from 'styled-components'
-import { ToastContainer, toast } from 'react-toastify';
-import 'react-toastify/dist/ReactToastify.css';
-import { connect } from 'react-redux'
-import { axiosWithBase } from '../../AxiosCustom'
-import * as actions from '../../store/Actions/actionCreators'
-
-import Spinner from '../Spinner'
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import { connect } from "react-redux";
+import { axiosWithBase } from "../../AxiosCustom";
+import * as actions from "../../store/Actions/actionCreators";
+import queryString from "query-string";
+import jwt from "jsonwebtoken";
```

Comment:
jsonwebtokens wasn't added to the dependencies in package.json so running `npm install` doesn't work when pulling this branch, each person will have to run `npm i jsonwebtoken` ...would be good if we could have it added to the package.json for ease in the future.